app.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+var FizzBuzz = (function() {
+  var _fizzBuzz = function() {
+    this.name = "FizzBuzz";
+    this.nums = [];
+  };
+
+  _fizzBuzz.prototype = {
+    getNums: function() {
+
+      if (this.nums.length > 0) {
+        this.nums.length = 0;
+      }
+
+      var low = parseInt(document.getElementById("low").value);
+      var high = parseInt(document.getElementById("high").value);
+
+      this.nums.push(low);
+      this.nums.push(high);
+
+    return this.nums;
+    },
+
+    renderFB: function() {
+      var input, output, writeNums, writeFB;
+
+      input = this.nums;
+      output = "";
+
+      writeNums = document.getElementById("displayNums");
+      writeNums.innerHTML = "";
+      writeNums.innerHTML = "<em>You chose the numbers " + input[0] + " and " + input[1] + ".</em>";
+
+      for (var i = input[0]; i <= input[1]; i++) {
+        if ((i % 3 === 0) && (i % 5 === 0)) {
+          output += "<span class='fizzbuzz'/>FizzBuzz</span> ";
```

Comment:
What's up with `"<span class='fizzbuzz'/>`? `span` is not a self-closing tag, and there's no need to escape anything here.